test/utils.ts

```diff
@@ -258,3 +263,75 @@ export function getLocalEip712Signature(
     // @ts-ignore
     return sigUtil.signTypedData_v4(privateKey, { data: typedRequestData });
 }
+
+export async function deployHub(
+    penalizer: string = constants.ZERO_ADDRESS,
+    configOverride: Partial<RelayHubConfiguration> = {}
+): Promise<RelayHubInstance> {
+    const relayHubConfiguration: RelayHubConfiguration = {
+        ...defaultEnvironment.relayHubConfiguration,
+        ...configOverride
+    };
+    return await RelayHub.new(
+        penalizer,
+        relayHubConfiguration.maxWorkerCount,
+        relayHubConfiguration.minimumEntryDepositValue,
+        relayHubConfiguration.minimumUnstakeDelay,
+        relayHubConfiguration.minimumStake
+    );
+}
+
+export async function getGaslessAccount(): Promise<AccountKeypair> {
+    const a = ethWallet.generate();
+    const gaslessAccount = {
+        privateKey: a.getPrivateKey(),
+        // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
why do we need to disable this line?

Reply:
We don't need indeed. I removed this line.